### PR TITLE
Added Cars to vehicles list

### PR DESCRIPTION
**Pull request by MikeParmese**
Added cars, primarily from:
https://newatlas.com/worlds-fastest-production-car-history/46175/

### CODE REVIEW

File: fbl31h/src/config/vehicles.json [hunk 1]
```diff
@@ -1,29 +1,43 @@
...
+    { "make": "Ferrari", "model": "GT4 Berlinetta Boxer", "topSpeed": 175, "year": 1970 },
+    { "make": "Lamborghini", "model": "Countach LP400", "topSpeed": 179.0, "year": 1974 },
     { "make": "Lamborghini", "model": "Countach LP500 S", "topSpeed": 182, "year": 1982 },
     { "make": "Ruf", "model": "BTR", "topSpeed": 190, "year": 1983 },
+    { "make": "Ferrari", "model": "288 GTO", "topSpeed": 188.0, "year": 1984 },
     { "make": "Porsche", "model": "959", "topSpeed": 198, "year": 1986 },
     { "make": "Ruf", "model": "CTR", "topSpeed": 213, "year": 1987 },
+    { "make": "Ferrari", "model": "F40", "topSpeed": 202, "year": 1988 },
+    { "make": "Vector", "model": "W8", "topSpeed": 242, "year": 1990 },
+    { "make": "Bugatti", "model": "EB110 GT", "topSpeed": 209, "year": 1991 },
```
Comment (kherge) [inline fbl31h/src/config/vehicles.json:34]: Source? Not in referenced article.

File: fbl31h/src/config/vehicles.json [hunk 2]
```diff
@@ -1,29 +1,43 @@
...
-    { "make": "Mercedes", "model": "Simplex", "topSpeed": 60, "year": 1900 },
-    { "make": "Austro-Daimler", "model": "Prince Henry", "topSpeed": 85, "year": 1910 },
-    { "make": "Duesenberg", "model": "Model J", "topSpeed": 119, "year": 1920 },
-    { "make": "Duesenberg", "model": "Model SJ", "topSpeed": 140, "year": 1930 },
-    { "make": "Ferrari", "model": "GT4 Berlinetta Boxer", "topSpeed": 175, "year": 1970 },
-    { "make": "Ferrari", "model": "F40", "topSpeed": 202, "year": 1980 },
-    { "make": "Hennessey", "model": "Venom GT", "topSpeed": 270, "year": 2011 }
-]
\ No newline at end of file
+    { "make": "Bugatti", "model": "Chiron", "topSpeed": 304.7, "year": 2019 }
```
Comment (kherge) [inline fbl31h/src/config/vehicles.json:42]: Source? Not in referenced article.

Comment (MikeParmese) [reply fbl31h/src/config/vehicles.json:42]: I believe all of the cars not in the referenced article came from this article:

https://www.topgear.com/car-news/supercars/bugattis-300mph-chiron-vs-fastest-cars-history#1

I should have included it yesterday as a referenced article

**Comment by kherge** [comment]
Thanks for the new vehicles!

**Pull request merged**